### PR TITLE
Lower timeout and disable retries for trace submission

### PR DESCRIPTION
**Pull request by hansihe**
I believe recurring issues with apps starting when the DD agent goes down is due to memory filling up with trace data.

This PR:
* Disables retries for trace submission. The DD agent should generally be on the same machine as the app, I don't believe there is any value in retrying submission of traces.
* Significantly lowers the timeouts for opening connections, waiting for the pool and receiving a response.

If submitting traces fails, the code already handles this gracefully by logging an error and proceeding. This is the behavior we want.

### CODE REVIEW

**Review by eva-marques — APPROVED** [review]

File: mix.exs [hunk 1]
```diff
@@ -2,7 +2,7 @@ defmodule SpandexDatadog.MixProject do
   use Mix.Project
 
   @source_url "https://github.com/surgeventures/spandex_datadog"
-  @version "1.6.4"
+  @version "1.7.0"
```
Comment (kamilkowalski) [inline mix.exs]: Technically this is a fix I guess.

**Review by kamilkowalski — APPROVED** [review]

**Comment by andreyuhai** [comment]
Could we publish this on Hex as well please?

**Comment by andreyuhai** [comment]
Oh I see the release [instructions](https://github.com/surgeventures/spandex_datadog/blob/master/RELEASEME.md) but I don't have the rights to publish.

**Comment by hansihe** [comment]
Oh, I thought this was one of the repos with auto publish set up

**Pull request merged**